upgrade_tests/test_scenarios/test_subscription.py

```diff
@@ -0,0 +1,106 @@
+"""Test for subscription manifest related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: CLI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from automation_tools.satellite6 import hammer
+from fabric.api import env, execute
+from nailgun import entities
+from unittest2.case import TestCase
+from upgrade_tests import post_upgrade, pre_upgrade
+from upgrade_tests.helpers.scenarios import get_satellite_host, upload_manifest
+import os
+
+
+class Scenario_manifest_refresh(TestCase):
+    """The test class contains pre-upgrade and post-upgrade scenarios to test
+    manifest refresh before and after upgrade
+
+    Test Steps:
+
+    1. Before Satellite upgrade, upload a manifest.
+    2. Refresh the manifest.
+    3. Upgrade satellite.
+    4. Refresh the manifest.
+    5. Delete the manifest.
+
+    """
+    manifest_url = os.environ.get('MANIFEST_URL')
```

Comment:
From where are you getting this ? And Can you rename this to UPGRADE_MANIFEST_URL ?

Reply:
We will get this from from gitlab config (see PR 153).